docs/data/sdks/typescript-react-native/index.md

```diff
@@ -311,6 +311,14 @@ import { flush } from '@amplitude/analytics-react-native';
 flush();
 ```
 
+By default, `flush` is called automatically in an interval, if you want to flush the events all together, you can control the async flow with the optional Promise interface, example:
+
+```typescript
```

Comment:
ts

Reply:
Thanks for reviewing! I tested locally, think `ts` and `typescript` are the same, I follow the same convention in the same file.

Reply:
yes either is fine